sass/themes/frost/components/_modal.scss

```diff
@@ -1,35 +1,41 @@
+// ==========================================================================
+// Modal
+// ==========================================================================
 .modal__overlay {
```

Comment:
Modal should align with form, has no slider and with no blue dots in the middle;

![u16 modal](https://user-images.githubusercontent.com/5208251/26925675-f98720a2-4c4a-11e7-902f-bbd9e64d8641.png)